server/migrations/v2.17.3-share-add-isdownloadable.js

```diff
@@ -0,0 +1,32 @@
+'use strict'
+
+const { DataTypes } = require('sequelize')
+
+/**
+ * @typedef MigrationContext
+ * @property {import('sequelize').QueryInterface} queryInterface - a Sequelize QueryInterface object.
+ * @property {import('../Logger')} logger - a Logger object.
+ *
+ * @typedef MigrationOptions
+ * @property {MigrationContext} context - an object containing the migration context.
+ */
+
+/**
+ * This migration script adds the isDownloadable column to the mediaItemShares table.
+ *
+ * @param {MigrationOptions} options - an object containing the migration context.
+ * @returns {Promise<void>} - A promise that resolves when the migration is complete.
+ */
+module.exports = {
+  up: async ({ context: { queryInterface, logger } }) => {
+    await queryInterface.addColumn('mediaItemShares', 'isDownloadable', {
```

Comment:
Since the `mediaItemShares` table was added later someone may be upgrading from a version that doesn't have the table yet which will cause this to crash. See `v2.17.3-fk-constraints` migration on wrapping this in a `tableExists` check.

Also, we want these migrations to be idempotent. If this migration were to run and the `isDownloadable` column already exists, the migration should still run.